src/tools/publish_toolstate.py

```diff
@@ -86,7 +86,7 @@ def gh_url():
     return os.environ['TOOLSTATE_ISSUES_API_URL']
 
 
-def maybe_delink(message):
+def maybe_unlink(message):
```

Comment:
This one I'm not sure about. "unlink" makes me think there's an existing link that's being removed, this is about a pending message that hasn't been posted yet.

Reply:
Isn't that what it does? Removes `@...` links in a github message

Reply:
Let's name this function `maybe_remove_ping` to avoid the ambiguity.